src/components/SearchBar.jsx

```diff
@@ -0,0 +1,41 @@
+import { InputLabel, Input, InputAdornment } from '@mui/material';
+import { Search, HighlightOff } from '@mui/icons-material';
+
+export function SearchBar({ searchTerm, setSearchTerm }) {
+	const onSearch = (e) => {
+		setSearchTerm(e.target.value);
+	};
+
+	const onSearchReset = () => {
+		setSearchTerm('');
+	};
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+	};
+
+	return (
+		<form onSubmit={handleSubmit}>
+			<InputLabel htmlFor="searchTerm">Filter Items</InputLabel>
+			<Input
+				sx={{ width: 350 }}
+				id="searchTerm"
+				value={searchTerm}
+				placeholder="Start typing here..."
+				onChange={onSearch}
+				startAdornment={
+					<InputAdornment position="start">
+						<Search />
+					</InputAdornment>
+				}
+				endAdornment={
+					searchTerm.length ? (
+						<InputAdornment position="end">
+							<HighlightOff onClick={onSearchReset} type="reset" />
+						</InputAdornment>
+					) : null
```

Comment:
I haven't used Material UI, but I really like this logic! I just notice that when you hoover over the HighlightOff icon, the cursor doesn't switch to an arrow, so a user might be confused if they can even click on that icon. If you have the time and bandwidth, see if you can add that arrow cursor. Other than that great job!